migrations/2023-11-06-153840_introduce_new_attempt_and_intent_status/down.sql

```diff
@@ -0,0 +1,12 @@
+-- This file should undo anything in `up.sql`
+DELETE FROM pg_enum
+WHERE enumlabel = 'partially_captured_and_capturable'
+AND enumtypid = (
+  SELECT oid FROM pg_type WHERE typname = 'IntentStatus'
+);
+
+DELETE FROM pg_enum
+WHERE enumlabel = 'partial_charged_and_chargeable'
+AND enumtypid = (
+  SELECT oid FROM pg_type WHERE typname = 'AttemptStatus'
+)
```

Comment:
Deleting enum values will require privileges 
```suggestion
SELECT 1
```